artifact/parser_test.cpp

```diff
@@ -39,6 +40,8 @@ class ParserTestEnv : public testing::Test {
 public:
 protected:
 	static void SetUpTestSuite() {
+		mender::common::log::SetLevel(mender::common::log::LogLevel::Trace);
```

Comment:
Hmm, I don't agree here actually. I think it should always be enabled. However, it should only be shown on a failing test, which is more up to the test framework I would think.

But if ur adamant...

Reply:
Not disagreeing with that! But right now it will be noisy. And even if we want to improve the test framework, it would be better to enable Trace level globally, not here in this single test, or?

Reply:
If you give the green light, I'm on it asap!

Reply:
As long as it's hidden for passing tests, I'm all for it. I'd recommend a new PR for this though, it's not really related to this code.